test/recipes/80-test_cms.t

```diff
@@ -760,6 +760,24 @@ subtest "CMS Check that bad attributes fail when verifying signers\n" => sub {
     }
 };
 
+subtest "CMS Check that bad encryption algorithm fails\n" => sub {
+    plan tests => 1;
```

Comment:
I think this must be something like
`plan tests => disabled("des") || disabled("legacy") ? 0 : 1;`

Reply:
hm, no. the problem is somewhere else.

Reply:
Hmm. Actually no. This test is expected to fail and is deliberately testing an attempt to use an encryption algorithm that is not available because the legacy provider is not loaded. If the legacy provider is not available at all then the test still works. If DES support is not even compiled in then the test isn't really testing the right thing - but it still fails and so it still passes.